deploy/template.yaml

```diff
@@ -583,6 +608,11 @@ Resources:
 
   IssueClientAccessTokenFunction:
     Type: AWS::Serverless::Function
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - W1001 # Ref to resource "IPVCoreInternalTestingApi" that may not be available when condition "IsDevelopment" is False
```

Comment:
This is a bit sad 😞 - I guess it's not clever enough to realise that the condition is the same

(I assume you've checked that it _isn't_ a problem - not actually sure which order it'll resolve everything!)

Reply:
I've not deployed to a non-dev environment to test it. But the validation failure breaks our existing workflows which is annoying. I think it's to do with how SAM gets transformed and generates new resources.